src/Client/BasicClient/BasicClientWithOnlineImpl.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TelegramOSINT\Client\BasicClient;
+
+use TelegramOSINT\MTSerialization\AnonymousMessage;
+use TelegramOSINT\TLMessage\TLMessage\ClientMessages\update_status;
+
+class BasicClientWithOnlineImpl extends BasicClientImpl
```

Comment:
Rename to BasicClientWithStatusReportingImpl

Reply:
Done